modules/Record.js

```diff
@@ -54,7 +54,8 @@ export default class Record {
     let result = {};
 
     for (let key in this.constructor[defaults]) {
-      result[key] = this[key];
+      let value = this[key]
+      result[key] = (value && value.toJSON) ? value.toJSON() : value
```

Comment:
Let's test for `typeof value.toJSON === 'function'`, to make sure that the call is possible.

Reply:
Also, don't forget semicolons :)